tests/integration/common.rs

```diff
@@ -149,8 +150,16 @@ impl Zizmor {
             OutputMode::Both => [output.stderr, output.stdout].concat(),
         })?;
 
+        let input_placeholder = "@@INPUT@@";
+        let input_path_regex = Regex::new(&format!(r"{input_placeholder}[\\/\w.-]+"))?;
         for input in &self.inputs {
-            raw = raw.replace(input, "@@INPUT@@");
+            raw = raw.replace(input, input_placeholder);
+            // Normalize Windows '\' file paths to using '/'
+            raw = input_path_regex
+                .replace_all(&raw, |captures: &Captures| {
+                    captures.get(0).unwrap().as_str().replace("\\", "/")
+                })
+                .into_owned();
```

Comment:
Out of curiosity, which of the `e2e` tests is revealing this behavior?

Reply:
This one if I recall correctly:
https://github.com/woodruffw/zizmor/blob/d3b7eb86c15d17533d9776fd2a5e485459268f81/tests/integration/snapshots/integration__e2e__menagerie.snap#L10-L13

On Windows you will get something like (with `\`):
```
INFO audit: zizmor: 🌈 completed @@INPUT@@\.github\dummy-action-2\action.yml 
...
```

Reply:
Got it, thanks!

I thought about it a bit, and I think we should put this in a a `cfg!` block that matches only Windows -- it probably doesn't affect anything anywhere else, but doing that will make it clearer to future contributors (including myself, who will forget) what's going on here 🙂